frontend/src/main.tsx

```diff
@@ -1,10 +1,11 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.tsx'
-import './index.css'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import React from 'react';
```

Comment:
Entirely up to you, I'd recommend disabling the rule that requires you to import React, intent is more easily communicated if you just `import { useEffect, useState } from 'react'` than the entire package where it's needed

Reply:
I have tried configuring disabling the rule that requires to import React, however, I'm failing to get the solution for this. Please check my eslint file for additions. Thank you, you're input is a lifesaver.

Reply:
Just write a new ticket for it and come back later. I wouldn't let it block this ticket considering its about pinging your API